src/components/Tool/FixPhoneNumbers/ContactPhoneNumbers.tsx

```diff
@@ -0,0 +1,231 @@
+  },
+}));
+
+interface Props {
+  index: number;
+  person: PersonInvalidNumberFragment;
+  phoneNumber: PhoneNumber;
+  handleChangePrimary: (phoneNumberId: string) => void;
+  handleDeleteNumberOpen: (person: {
+    personId: string;
+    phoneNumber: PhoneNumber;
+  }) => void;
+  handleSingleConfirm: (
+    person: PersonInvalidNumberFragment,
+    numbers: PhoneNumber[],
+  ) => void;
+  errors: any;
+}
+
+export const ContactPhoneNumbers: React.FC<Props> = ({
+  errors,
+  index,
+  person,
+  phoneNumber,
+  handleChangePrimary,
+  handleDeleteNumberOpen,
+}) => {
+  const { classes } = useStyles();
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const { id: personId } = person;
+
+  return (
+    <>
+      <Grid
+        data-testid="phoneNumbers"
+        item
+        xs={6}
+        sm={4}
+        className={classes.paddingB2}
+      >
+        <Box
+          display="flex"
+          justifyContent="space-between"
+          className={classes.paddingX}
+        >
+          <Box>
+            <Hidden smUp>
+              <Typography display="inline" variant="body2" fontWeight="bold">
+                {t('Source')}:
+              </Typography>
+            </Hidden>
+            <Typography display="inline" variant="body2">
+              {`${sourceToStr(t, phoneNumber?.source)} (${dateFormatShort(
+                DateTime.fromISO(phoneNumber?.updatedAt),
+                locale,
+              )})`}
+            </Typography>
+          </Box>
+        </Box>
+      </Grid>
+      <Grid item xs={6} sm={2} className={classes.paddingB2}>
+        <Box
+          display="flex"
+          justifyContent="center"
+          className={classes.paddingX}
+        >
+          <Typography display="flex" alignItems="center">
+            {phoneNumber?.primary ? (
+              <>
+                <Hidden smUp>
+                  <Typography
+                    display="inline"
+                    variant="body2"
+                    fontWeight="bold"
+                  >
+                    {t('Source')}:
+                  </Typography>
+                </Hidden>
+                <StarIcon
+                  data-testid={`starIcon-${personId}-${phoneNumber?.id}`}
+                  className={classes.hoverHighlight}
+                />
+              </>
+            ) : (
+              <>
+                <Hidden smUp>
+                  <Typography
+                    display="inline"
+                    variant="body2"
+                    fontWeight="bold"
+                  >
+                    {t('Source')}:
+                  </Typography>
+                </Hidden>
+                <Tooltip title={t('Set as Primary')} placement="left">
+                  <StarOutlineIcon
+                    data-testid={`starOutlineIcon-${personId}-${phoneNumber?.id}`}
+                    className={classes.hoverHighlight}
+                    onClick={() => handleChangePrimary(phoneNumber?.id)}
```

Comment:
Same here as well. It looks like all the properties on phoneNumber have optional chaining.